Partly fix issue Document missing Mixxx Controls #261 (#348)
mixxxdj/manual: source/chapters/advanced_topics.rst

   .. versionadded:: 2.3.0

.. mixxx:control:: [ChannelN],cue_cdj
                   [PreviewDeckN],cue_cdj
                   [SamplerN],cue_cdj

   Represents a :guilabel:`Cue` button that is always in :term:`CDJ` mode.

   :range: binary
   :feedback: None

   .. versionadded:: 1.10.0

.. mixxx:control:: [ChannelN],cue_clear
                   [PreviewDeckN],cue_clear
                   [SamplerN],cue_clear

>    Releases the already set cue point and sets cue value to 0.

Reviewer: ```suggestion
   Deletes the already set cue point and sets :mixxx:coref:`[ChannelN],cue` to 0.
```

>    Deletes the already set cue point and sets :mixxx:coref:`[ChannelN],cue` to 0.

Reviewer: ```suggestion
   Deletes the already set cue point and sets :mixxx:coref:`[ChannelN],cue_point` to -1.
```

                   [PreviewDeckN],cue_indicator
                   [SamplerN],cue_indicator

>    Provides information to be bound to the :guilabel:`Cue` Button e.g. blinking when next press will move the :term:`cue point`.

Reviewer: ```suggestion
   Indicates the blinking pattern of the :guilabel:`CUE` button (i.e. 1.0 if the button is illuminated, 0.0 otherwise), depending on the chosen :mixxx:coref:`cue mode <[ChannelN],cue_mode>`.
```

   :range: binary
   :feedback: :guilabel:`Cue` button

   .. versionadded:: 2.0.0

.. mixxx:control:: [ChannelN],cue_mode
                   [PreviewDeckN],cue_mode
                   [SamplerN],cue_mode

>    Decides which cue mode to call based on the user preference and compatible hardware.

Reviewer: ```suggestion
   Represents the currently chosen :ref:`cue mode <interface-cue-modes>`.
```

   .. versionadded:: 2.3.0

> The ``[Broadcast]`` controls

Reviewer: ```suggestion
The ``[Shoutcast]`` controls
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. mixxx:control:: [Shoutcast],enabled

>    This control shows if live Internet broadcasting is enabled with Shoutcast server connection. :term:`Shoutcast <shoutcast>` feeds to and from the stream.

Reviewer: ```suggestion
   Shows if live Internet broadcasting is enabled.
```

   .. versionadded:: 2.3.0

.. mixxx:control:: [Controls],ShowDurationRemaining

>    Yields the current state of the remaining time duration of the loaded track.

Reviewer: ```suggestion
   Represents the current state of the remaining time duration display of the loaded track.
```
